.travis.xacro_test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
ah, yes. that actually makes sense..
Did you declare a dependency on rosbash? ;)

```
ADDITIONAL_DEBS='ros-$ROS_DISTRO-rosbash'
```